content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,295 @@
+
+As someone setting up authentication and authorization on your production Kubernetes cluster, here are some things to consider:
+
+- *Set the authorization mode*: When the Kubernetes API server
+([kube-apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/))
+starts, the supported authentication modes must be set using the *--authorization-mode*
+flag. For example, that flag in the *kube-adminserver.yaml* file (in */etc/kubernetes/manifests*)
+could be set to Node,RBAC. This would allow Node and RBAC authorization for authenticated requests.
+- *Create user certificates and role bindings (RBAC)*: If you are using RBAC
+authorization, users can create a CertificateSigningRequest (CSR) that can be
+signed by the cluster CA. Then you can bind Roles and ClusterRoles to each user.
+See [Certificate Signing Requests](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/)
+for details.
+- *Create policies that combine attributes (ABAC)*: If you are using ABAC
+authorization, you can assign combinations of attributes to form policies to
+authorize selected users or groups to access particular resources (such as a
+pod), namespace, or apiGroup. For more information, see
+[Examples](https://kubernetes.io/docs/reference/access-authn-authz/abac/#examples).
+- *Consider Admission Controllers*: Additional forms of authorization for
+requests that can come in through the API server include
+[Webhook Token Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#webhook-token-authentication).
+Webhooks and other special authorization types need to be enabled by adding
+[Admission Controllers](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/)
+to the API server.
+
+## Production workload resources
+
+When planning a production Kubernetes cluster, you need to prepare for the scale and nature of your workloads. Demands from workloads can cause pressure outside of the Kubernetes control plane as well. Here are a few things to consider when managing workload resources:
+
+- *Set namespace limits*: To manage defaults and the limits of user workloads,
+you can set per-namespace quotas on things like memory and CPU. See
+[Manage Memory, CPU, and API Resources](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/)
+for details. You can also set
+[Hierarchical Namespaces](https://kubernetes.io/blog/2020/08/14/introducing-hierarchical-namespaces/)
+for inheriting limits.
+- *Consider horizontal pod autoscaling*: In preparation for demands on workloads in the
+cluster, you can use pod autoscaling to automatically add additional pods as needed.
+New pods are created based on an observation that a pod is exceeding its CPU
+utilization or other metric provided by the application. See
+[Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+for details.
```

Comment:
These are more about making your _workload_ production ready. I think that's important but should be the topic of a separate page.

Think [CKAD](https://www.cncf.io/certification/ckad/) vs [CKA](https://www.cncf.io/certification/cka/).

Reply:
Good point. I'll remove it from here and look for an opportunity to put this content elsewhere.